dask/tests/test_distributed.py

```diff
@@ -130,8 +130,20 @@ def test_futures_to_delayed_array(c):
     assert_eq(A.compute(), np.concatenate([x, x], axis=0))
 
 
+# with pytest.warns(UserWarning, match=warning_message) as user_warnings_b:
+#     get_scheduler(scheduler="dask.distributed")
+#     get_scheduler(scheduler="distributed")
+#     get_scheduler()
+# assert len(user_warnings_b) == 0
```

Comment:
Was this intended to be added as a proper test?

Reply:
I'm sorry, I forgot to remove those comments. I just pushed a new version after removing them.